src/applications/personalization/rated-disabilities/actions/index.js

```diff
@@ -1,4 +1,5 @@
 import { getData } from '../util';
+import recordEvent from 'platform/monitoring/record-event';
```

Comment:
nit: I prefer putting `platform` imports above relative imports but below imports from `node_modules` but that isn't a rule we have.

Reply:
I'm a fan of ordering imports. In a previous job there was a linting rule for enforcing alphabetical order haha.